apps/dashboard/app/views/layouts/editor.html.erb

```diff
@@ -9,7 +9,7 @@
 
   <%= javascript_include_tag 'application', nonce: true %>
   <%= javascript_include_tag 'editor', nonce: true %>
-  <%= stylesheet_link_tag 'application', nonce: true, media: 'all' %>
+  <%= stylesheet_link_tag 'application', nonce: content_security_policy_nonce, media: 'all', rel: 'preload stylesheet', as: 'style', type: 'text/css' %>
```

Comment:
We don't really need this so much now, but it can't hurt. In any case - this we do _need_ to actually use the `nonce` correctly here.  `javascript_include_tag(nonce: true)` works like you'd expect - giving a nonce, but `stylesheet_link_tag` does not so we need to actually set the nonce to the correct value and not just use `true`.